socorro/external/es/connection_context.py

```diff
@@ -90,6 +89,9 @@ def __init__(self, config):
         super().__init__()
         self.config = config
 
+        # Cached answers for things that don't change
+        self._mapping_cache = {}
```

Comment:
If we find ourselves doing this much more, we should write up a `methodcache`. I didn't do that, yet, because I didn't want to deal with locking and all the other complexities involved in caching on the instance.